### PR TITLE
Add feature status page for pluggable key/cert support for Istio CA [Stable]

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by brian-avery** [comment]
@liminw can you please take a look at this?

**Comment by brian-avery** [comment]
@istio/wg-security-maintainers can you please review?

**Comment by johnma14** [comment]
@liminw @istio/wg-security-maintainers gentle reminder for a quick review. Thanks.

File: features/pluggable-key-cert-istio-ca.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+**Feature name:**
+
+Pluggable Key/Cert Support for Istio CA
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+Limin Wang
```
Comment (liminw) [inline features/pluggable-key-cert-istio-ca.md:23]: Oliver should be the lead for this feature. @myidpt Can you comment on the PR?

**Comment by johnma14** [comment]
Thanks @liminw. Made the changes.

File: features/pluggable-key-cert-istio-ca.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+**Feature name:**
+
+Pluggable Key/Cert Support for Istio CA
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+Oliver Liu [myidpt]
```
Comment (myidpt) [inline features/pluggable-key-cert-istio-ca.md:23]:
```suggestion
Oliver Liu [myidpt] and Lei Tang [lei-tang]
```

File: features/pluggable-key-cert-istio-ca.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Stable
+
+### Requirements: 
+
+**Performance**
+
+- [ ] Latency, throughput, and scalability are quantified and documented on
```
Comment (myidpt) [inline features/pluggable-key-cert-istio-ca.md:202]:
```suggestion
- [//] Latency, throughput, and scalability are quantified and documented on
```

File: features/pluggable-key-cert-istio-ca.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+	istio.io. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues.
```
Comment (myidpt) [inline features/pluggable-key-cert-istio-ca.md:207]:
```suggestion
- [//] Feature has no known major issues. 
```

File: features/pluggable-key-cert-istio-ca.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+- [ ] Latency, throughput, and scalability are quantified and documented on
+	istio.io. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues. 
+
+**Approvals**
+
+- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
```
Comment (myidpt) [inline features/pluggable-key-cert-istio-ca.md:211]:
```suggestion
- [//] The appropriate work group(s) have reviewed and approved promotion of the feature.
```

File: features/pluggable-key-cert-istio-ca.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+- [ ] The [supportability review panel](https://docs.google.com/document/d/1w0epyFhhDSf_TwFEfa_lrn1v61mXNJKpEp_kUgp4sSc/edit#) has reviewed the feature in order to find any supportability concerns.  
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the
```
Comment (myidpt) [inline features/pluggable-key-cert-istio-ca.md:213]:
```suggestion
- [//] The TOC has reviewed and approved promotion of the feature as part of the
```

**Comment by johnma14** [comment]
@myidpt Thank you for the review. I made the recommended changes plus also marked checkboxes for alpha and beta. If you can take a look at the checkboxes for alpha and beta would be great. Just wanted to make sure I captured the correct testing and documentations applicable to the feature. Thank you.

**Review by myidpt — APPROVED** [review]

**Pull request closed without merging**